eng/pipelines/common/variables.yml

```diff
@@ -10,9 +10,9 @@ variables:
   value: 1000
 
 - name: isOfficialBuild
-  value: ${{ and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}
+  value: ${{ and(eq(variables['System.TeamProject'], 'internal'), eq(variables['Build.DefinitionName'], 'dotnet-runtime-official')) }}
```

Comment:
What def are we using internal PRs too? Probably this should also have !pr checks .

Reply:
We are using one called `runtime` but I think it is a good idea to be defensive and consider PR reason.